.github/workflows/vast.yaml

```diff
@@ -535,6 +541,13 @@ jobs:
         with:
           python-version: "3.9"
 
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - run: |
+          corepack enable
```

Comment:
In all instances of `corepack enable` in the CI we should give the step a titlecased name for consistency with the other steps.